gqlalchemy/how-to-guides/query-builder.md

```diff
@@ -0,0 +1,1498 @@
+
+### Set a property
+
+To **set a property** of a graph object use the **assignment operator** from the query builder or a simple equals sign as a string - `"="`.
+
+<Tabs
+  defaultValue="gqlalchemy"
+  values={[
+    {label: 'GQLAlchemy', value: 'gqlalchemy'},
+    {label: 'Cypher', value: 'cypher'}
+  ]}>
+  <TabItem value="gqlalchemy">
+
+```python
+from gqlalchemy import match
+from gqlalchemy.query_builders.memgraph_query_builder import Operator
+
+query = (
+    create()
+    .node(labels="Country", variable="c", name="Germany")
+    .set_(item="c.population", operator=Operator.ASSIGNMENT, literal=83000001)
+    .execute()
+)
+```
+
+  </TabItem>
+  <TabItem value="cypher">
+
+```cypher
+CREATE (c:Country {name: 'Germany'}) SET c.population = 83000001;
+```
+
+</TabItem>
+</Tabs>
+
+:::info
+`Operator` is an enumeration class defined in the
+[`declarative_base.py`](https://github.com/memgraph/gqlalchemy/blob/main/gqlalchemy/query_builders/declarative_base.py#L84-L94). It can be imported from `gqlalchemy.query_builders.memgraph_query_builder`. 
+
+If you don't want to import it, you can use strings `"="`, `">="`, `">"`, `"<>"`, `":"`, `"<"`, `"<="`, `"!="` or `"+="` instead.
+:::
+
+To **set a property of already existing node**, first match the node and then set its property. 
+
+<Tabs
+  defaultValue="gqlalchemy"
+  values={[
+    {label: 'GQLAlchemy', value: 'gqlalchemy'},
+    {label: 'Cypher', value: 'cypher'}
+  ]}>
+  <TabItem value="gqlalchemy">
+
+```python
+from gqlalchemy import match
+from gqlalchemy.query_builders.memgraph_query_builder import Operator
+
+query = (
+    match()
+    .node(labels="Country", variable="c", name="Germany")
+    .set_(item="c.population", operator=Operator.ASSIGNMENT, literal=10000)
+    .execute()
+)
+```
+
+  </TabItem>
+  <TabItem value="cypher">
+
+```cypher
+MATCH (c:Country {name: 'Germany'}) SET c.population = 10000;
+```
+
+</TabItem>
+</Tabs>
+
+To **set multiple properties of a node**, run the following code:
+
+<Tabs
+  defaultValue="gqlalchemy"
+  values={[
+    {label: 'GQLAlchemy', value: 'gqlalchemy'},
+    {label: 'Cypher', value: 'cypher'}
+  ]}>
+  <TabItem value="gqlalchemy">
+
+```python
+from gqlalchemy import match
+from gqlalchemy.query_builders.memgraph_query_builder import Operator
+
+query = (
+    match()
+    .node(variable="n")
+    .where(item="n.name", operator="=", literal="Germany")
+    .set_(item="n.population", operator=Operator.ASSIGNMENT, literal=83000001)
+    .set_(item="n.capital", operator=Operator.ASSIGNMENT, literal="Berlin")
+    .execute()
+)
+```
+
+  </TabItem>
+  <TabItem value="cypher">
+
+```cypher
+MATCH (n) WHERE n.name = 'Germany' SET n.population = 83000001 SET n.capital = 'Berlin';
+```
+
+</TabItem>
+</Tabs>
+
+If a node already had the properties we are setting, they will be updated to a new value. Otherwise, the properties will be created and their value will be set.
```

Comment:
```suggestion
If a node already has the properties we are setting, they will be updated to a new value. Otherwise, the properties will be created and their value will be set.
```